Compiler/BackEnd/BackendInline.mo

```diff
@@ -665,9 +665,9 @@ function: inlineEquationArray
   input BackendDAE.EquationArray inEquationArray;
   input Inline.Functiontuple fns;
   input BackendDAE.Shared iShared;
-  output BackendDAE.EquationArray outEquationArray;
+  output BackendDAE.EquationArray outEquationArray = inEquationArray;
   output BackendDAE.EqSystem outEqs;
```

Comment:
If you do the assignment in the else cases instead of using a default value, then it should not take any memory for the `normal` cases, right?